azure-devops/azext_devops/dev/common/utils.py

```diff
@@ -0,0 +1,39 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+def open_file(filepath):
+    import subprocess
+    import platform
+    import os
+    if platform.system() == 'Darwin':       # macOS
+        subprocess.call(('open', filepath))
+    elif platform.system() == 'Windows':    # Windows
+        # disable pylint warning since we run pylint on linux agent and startfile is a windows only function
+        os.startfile(filepath)  # pylint: disable=no-member
```

Comment:
just confirming that this will handle appselector scenario also? where windows is opening yml file extension for first time and does not have any default app associated with it?

also this function will open file and wait for it to close, or just open and return?

Reply:
It will handle app selector by opening the choose app dialog. 
Function will open the file and exit, it does not wait. I will add a header with this info.